proto/frequenz/dispatch/v1/dispatch.proto

```diff
@@ -114,6 +114,60 @@ message DispatchComponentIDs {
   repeated uint64 component_ids = 1;
 }
 
+// Ruleset governing when and how a dispatch should re-occur
+message RecurrenceRule {
```

Comment:
This RecurrenceRule is nowhere applied. You need to add a new property in the Dispatch message itself I think.